icons/qwik/qwik-plain-wordmark.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <g>
+        <path style=" stroke:none;fill-rule:nonzero;fill:rgb(9.411765%,71.372549%,96.470588%);fill-opacity:1;" d="M 30.234375 80.96875 L 24.445312 75.210938 L 24.363281 75.222656 L 24.363281 75.164062 L 12.054688 62.988281 L 15.09375 60.058594 L 13.308594 49.816406 L 4.847656 60.308594 C 3.410156 61.761719 3.136719 64.132812 4.175781 65.875 L 9.464844 74.648438 C 10.273438 75.992188 11.542969 76.855469 13.316406 76.792969 C 17.066406 76.660156 18.71875 76.660156 18.71875 76.660156 L 30.230469 80.964844 Z M 30.234375 80.96875 "/>
+        <path style=" stroke:none;fill-rule:nonzero;fill:rgb(9.411765%,71.372549%,96.470588%);fill-opacity:1;" d="M 33.09375 64.832031 C 33.929688 63.113281 34.226562 61.609375 33.402344 60.097656 L 32.234375 57.945312 L 31.628906 56.839844 L 31.390625 56.40625 L 31.371094 56.433594 L 28.1875 50.910156 C 27.792969 50.226562 27.222656 49.65625 26.535156 49.261719 C 25.847656 48.871094 25.070312 48.667969 24.277344 48.675781 L 21.484375 48.753906 L 13.152344 48.777344 C 12.378906 48.78125 11.621094 48.988281 10.949219 49.375 C 10.28125 49.761719 9.722656 50.316406 9.332031 50.984375 L 4.269531 61.039062 L 13.328125 49.765625 L 25.214844 62.835938 L 23.085938 64.992188 L 24.359375 75.214844 L 24.375 75.195312 L 24.375 75.222656 L 24.355469 75.222656 L 24.382812 75.25 L 25.375 76.214844 L 30.164062 80.898438 C 30.367188 81.089844 30.695312 80.859375 30.558594 80.617188 L 27.597656 74.785156 "/>
```

Comment:
Plain icons should only have a single path. So please combine the paths if possible, :)